### PR TITLE
Translate minio docs to chinese

### PR DESCRIPTION
**Pull request by dingjs**
Translate minio docs to chinese

## Description
1. add README_zh_CN chinese simplified version
2. add minio-limitations.md in docs/zh_CN

## Motivation and Context
Minio is a great project,it should get more attention from chinese developers.I will finish the translation in october.

## How Has This Been Tested?
No change of the code.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: docs/zh_CN/FreeBSD.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Minio FreeBSD Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+### Minio with ZFS backend - FreeBSD
+This example assumes that you have a FreeBSD 11.x running
```
Comment (deekoder) [inline docs/zh_CN/FreeBSD.md]: Will this also be translated?

File: docs/zh_CN/admin-api/README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
+# Management REST API
+
+## Authentication
+- AWS signatureV4
+- We use "minio" as region. Here region is set only for signature calculation.
```
Comment (deekoder) [inline docs/zh_CN/admin-api/README.md:5]: @dingjs  will this doc be translated as well?

**Review by deekoder — CHANGES_REQUESTED** [review]
@dingjs  thank you for this amazing PR. Just wondering if some of the files in zh_CN folder which are still in english will be translated later.

File: docs/zh_CN/FreeBSD.md (hunk 1 above)
Comment (dingjs) [reply docs/zh_CN/FreeBSD.md]: Yes,right now I only translated two files,the others in zh_CN is still in english,I will finish the job in october.

File: docs/zh_CN/admin-api/README.md (hunk 2 above)
Comment (dingjs) [reply docs/zh_CN/admin-api/README.md:5]: All the files in zh_CN will be translated.If docs.minio.com has a zh_CN language option,maybe more chinese developers will participate and speed up the translation.

**Comment by harshavardhana** [comment]
@dingjs we have created a branch for you https://github.com/minio/minio/tree/chinese-translation please send the PRs for this branch instead. Thanks

**Comment by deekoder** [comment]
@dingjs if you can send this pr to that branch that would be great.

**Review by harshavardhana — APPROVED** [review]
Thanks this @dingjs . This LGTM @deekoder

**Comment by dingjs** [comment]
This PR has some commit conflicts with the others.I'm closing thid PR and reopen another one with no conflicts,only translate the docs in docs/zh_CN.

**Pull request closed without merging**